llvm/lib/Transforms/Utils/BuildLibCalls.cpp

```diff
@@ -554,6 +579,7 @@ bool llvm::inferNonMandatoryLibFuncAttrs(Function &F,
     Changed |= setDoesNotThrow(F);
     Changed |= setDoesNotCapture(F, 0);
     Changed |= setOnlyReadsMemory(F, 0);
+    Changed |= setOnlyAccessesErrnoMemory(F);
```

Comment:
IIUC it should be `memory(argmem: read, errnomem: write)`.

Reply:
Hmm, shouldn’t marking all pointer arguments as readonly be semantically equivalent to having ArgMem: Ref?